modelcontextprotocol/server.py

```diff
@@ -695,26 +695,32 @@ def create_glossary_categories(categories) -> List[Dict[str, Any]]:
 
 
 def main():
-    mcp.run()
-
-
-if __name__ == "__main__":
+    """Main entry point for the Atlan MCP Server."""
     parser = argparse.ArgumentParser(description="Atlan MCP Server")
     parser.add_argument(
         "--transport",
         type=str,
-        default="stdio",
+        default=os.getenv("MCP_TRANSPORT", "stdio"),
```

Comment:
We also use dotenv for env variables. Lets use the setting.py for these configurations